### DISCUSSION TITLE
How can ClearRefinements exclude query?

### DISCUSSION THREAD

**Discussion by JSShant** [algolia/instantsearch#5754] [Q&A] [answered]
Just as the above says:
Is it possible to not remove search query on Algolia Clear Filter as we cant include the search query into excluded attributes. Thankyou!

**Comment by Haroenv** [comment, marked as answer]
The default value of excludedAttributes is already ['query'], so it's not taken in account by default. Do you have a sandbox with the behaviour you don't expect?
